### ISSUE TITLE
Expressions on empty lines

### ISSUE THREAD

**Issue by vicchig** [MajeedKazemi/nova-editor#483]
We allow these types of modifiers to be inserted on empty lines. @MajeedKazemi  is that intended?

![image](https://user-images.githubusercontent.com/44914019/140608878-71abc076-2d70-4772-a64b-1064c4bea50b.png)

**Comment by vicchig** [comment]
I guess we can allow them, but it would probably be good to add a warning to them saying that the statement has no effect.

**Comment by MajeedKazemi** [comment]
hmm, I didn't know that we could do these kind of insertions. I personally believe that this should not happen.

**Comment by vicchig** [comment]
Yes, same. What about variable references on empty lines? I think allowing those is ok and draft mode makes sense for them as well. Those can always be turned to at least an assignment statement.
